.travis.yml

```diff
@@ -6,6 +6,11 @@ node_js:
 services:
   - docker
   - rabbitmq
+  - postgresql
+
+before_script:
```

Comment:
My PR moves all tests over to docker containers, so we won't have to worry about this too much.